website/content/en/preview/upgrade-guide.md

```diff
@@ -41,6 +41,8 @@ Karpenter ships with a few Custom Resource Definitions (CRDs). These CRDs are pu
     ```
     helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version vx.y.z --namespace karpenter --create-namespace
     ```
+* Follow [Troubleshooting](./troubleshooting.md#helm-error-when-upgrading-from-older-karpenter-version) when getting `invalid ownership metadata; label validation error:` after `helm upgrade` from older karpenter versions.
```

Comment:
```suggestion
{{% alert title="Note" color="warning" %}}
Follow [Troubleshooting]({{<ref "./troubleshooting#helm-error-when-upgrading-from-older-karpenter-version" >}}) when getting `invalid ownership metadata; label validation error:` after `helm upgrade` from older karpenter versions.
{{% /alert %}}
```

The old link leads to a 404. Also, I think this callout should be good as a Note. Make sure to change this everywhere.